squirrel.api/src/main/java/org/dice_research/squirrel/metadata/CrawlingActivity.java

```diff
@@ -134,6 +144,7 @@ public Model prepareMetadataModel() {
 
         if (workerUri != null) {
             model.add(activity, PROV_O.wasAssociatedWith, model.createResource(workerUri));
+            model.add(generatedUris, PROV_O.wasAttributedTo, model.createResource(workerUri));
```

Comment:
As per the ontology diagram shown [here](https://www.w3.org/TR/2013/REC-prov-o-20130430/#description-starting-point-terms), we have defined generatedURIs as an entity. As per the diagram, an entity will be generated by an Activity and will be attributed to an Agent. 

Since, in our case, activity is squirrel activity and agent is the worker, we have used them respectively. generatedURIs agent is generated by squirrel activityUri and is attributed to workerUri.

Reply:
You are right. :+1: